GPU/GPUTracking/Base/GPUReconstructionKernels.h

```diff
@@ -31,19 +31,19 @@ GPUCA_KRNL((GPUITSFitterKernel                           ), (),       (), ())
 GPUCA_KRNL((GPUTPCConvertKernel                          ), (),       (), ())
 GPUCA_KRNL((GPUTPCCompressionKernels,   step0attached    ), (),       (), ())
 GPUCA_KRNL((GPUTPCCompressionKernels,   step1unattached  ), (),       (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, fillChargeMap    ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, resetMaps        ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, findPeaks        ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, noiseSuppression ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, updatePeaks      ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, countPeaks       ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, computeClusters  ), (single), (), ())
-GPUCA_KRNL((GPUTPCClusterFinderKernels, nativeScanUpStart), (single), (, int iBuf, int stage), (, iBuf, stage))
-GPUCA_KRNL((GPUTPCClusterFinderKernels, nativeScanUp     ), (single), (, int iBuf, int nElems), (, iBuf, nElems))
-GPUCA_KRNL((GPUTPCClusterFinderKernels, nativeScanTop    ), (single), (, int iBuf, int nElems), (, iBuf, nElems))
-GPUCA_KRNL((GPUTPCClusterFinderKernels, nativeScanDown   ), (single), (, int iBuf, unsigned int offset, int nElems), (, iBuf, offset, nElems))
-GPUCA_KRNL((GPUTPCClusterFinderKernels, compactDigit     ), (single), (, int iBuf, int stage, GPUPtr1(deprecated::PackedDigit*, in), GPUPtr1(deprecated::PackedDigit*, out)), (, iBuf, stage, GPUPtr2(deprecated::PackedDigit*, in), GPUPtr2(deprecated::PackedDigit*, out)))
-GPUCA_KRNL((GPUTPCClusterFinderKernels, decodeZS         ), (single), (), ())
+GPUCA_KRNL((ChargeMapFiller,            fillChargeMap    ), (single), (), ())
```

Comment:
when you have only a single kernel per class, you don't need to give it a name, it will take the dafault kernel by default.
Could you prefix the cluster finder kernels with GPUTPCCF, to make it clear what it is?

Reply:
Renamed the classes. Is there a reason to using a prefix instead of moving everything into it's own namespace?

Reply:
I think a namespace would work as well, but perhaps I'd need to adjust these macros, since they produce the kernel names, and that will most likely lead to a problem with the :: in it. If you want, you can just try, or I can have a look. But for now, I'll just merge as is once the CI passes.

Reply:
Yeah sure. Just something i'll keep in mind for when i feel like refactoring again. :-)